smartling-contexts-api/src/main/java/com/smartling/api/contexts/v2/pto/AsyncProcessPTO.java

```diff
@@ -0,0 +1,21 @@
+package com.smartling.api.contexts.v2.pto;
+
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.smartling.api.v2.response.ResponseData;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class AsyncProcessPTO implements ResponseData
+{
+    private String processUid;
+    private AsyncProcessState state;
```

Comment:
I've changed these parameters to `processState` and `processType`

Reply:
done